admin_guide/upgrading_citus.rst

```diff
@@ -88,22 +88,26 @@ After installing the new package and restarting the database, run the extension
 
 .. _upgrading_postgres:
 
-Upgrading PostgreSQL version from 9.6 to 10.0
-#############################################
+Upgrading PostgreSQL version from 10 to 11
+##########################################
+
+.. note::
+
+   Do not attempt to upgrade *both* Citus and Postgres versions at once. If both upgrades are desired, upgrade Citus first. Older versions of Citus are not always compatible with the newest Postgres.
```

Comment:
Can we add some more specifics here? e.g.: Citus 7.x is *not* compatible with PG11. So to upgrade PG 10 to 11, follow the above steps <link to Citus major version upgrade> to first upgrade from Citus 7.x to 8.0. Once you have done so, then continue below.